public/pages/Overview/utils/helpers.ts

```diff
@@ -121,6 +123,21 @@ export function getVisualizationSpec(description: string, data: any, layers: any
   };
 }
 
+export const getYDomainRange = (data: any[], timeUnit: string): number[] => {
```

Comment:
We should write JSDoc comment above this method to explain why we need this?

Reply:
Added.